src/sanitize.js

```
@@ -24,56 +24,75 @@
 
-    // Escape special characters that could be used for SQL injection
+    // Thoát các ký tự đặc biệt có thể được sử dụng cho SQL injection
     let sanitized = input
-      .replace(/'/g, "''")           // Escape single quotes
-      .replace(/\\/g, "\\\\")        // Escape backslashes
-      .replace(/\0/g, "\\0")         // Escape null bytes
-      .trim();                       // Trim whitespace
+      .replace(/'/g, "''")           // Thoát dấu nháy đơn
+      .replace(/\\/g, "\\\\")        // Thoát dấu gạch chéo ngược
+      .replace(/\0/g, "\\0")         // Thoát byte null
+      .trim();                       // Cắt khoảng trắng
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)